common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -114,6 +114,18 @@ class CapaFields(object):
         help=_("Amount of time after the due date that submissions will be accepted"),
         scope=Scope.settings
     )
+    show_correctness = String(
+        display_name=_("Show Results"),
+        help=_("Defines when to show whether a learner's answer to the problem is correct. "
+               "Configured on the subsection. A default value for the course can be set in Advanced Settings."),
```

Comment:
@cahrens You're totally right.  My apologies!  I'll fix the help text once the bokchoy tests are passing for the previous change.